### PR TITLE
[Doc Improvement][RSC permissions for a team][3380152]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/28fcf823e0f3ebe52ab984d79e4cd34fbae4c976/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 97 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e4acc41-0c57-40e4-83b1-111db2c85db8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/rsc/resource-specific-consent.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Use RSC permissions to determine the data access methods for your app. A user's
 
 ### Basic RSC Permissions
 
-A limited set of RSC permissions have been reviewed by Microsoft privacy and security teams and have been deemed low risk. These permissions can be consented to at all times by any user when they install and use an app. The following low-risk basic permissions will always be consented to upon installation. 
+A limited set of RSC permissions have been reviewed by Microsoft privacy and security teams and have been deemed low risk. These permissions can be consented to at all times by any user when they install and use an app. The following low-risk basic permissions will always be consented to upon installation.
```
Comment (v-deabira) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:54]: Microsoft's privacy and security teams have evaluated a limited set of Resource-Specific Consent (RSC) permissions and identified them as low risk. Users can consent to these permissions at any time during the installation and usage of an app. The following basic permissions, deemed low-risk, are always consented to upon installation:

**Review by v-deabira — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0ce97aabd5ce4185ba2f0348e654cdd5e21829bf/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 98 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/db77f18e-4446-481e-99d7-3a48c1575862) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/rsc/resource-specific-consent.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/graph-api/rsc/resource-specific-consent.md]: Updated.

**Review by Wajeed-msft — APPROVED** [review]

File: msteams-platform/graph-api/rsc/resource-specific-consent.md [hunk 2]
```diff
@@ -101,6 +101,7 @@ The following table provides RSC application permissions for a team and their ap
 |`Owner.Read.Group`|Read this group's owners.| NA | Supported |
 |`TeamsActivity.Send.Group`|Send activity feed notifications to users in this team. | NA | Supported |
 |`TeamsAppInstallation.Read.Group`|Read the apps that are installed in this team.| NA | Supported |
+|`TeamsAppInstallation.ReadWriteAndConsentSelfForChat.All`|Allows a Teams app to read, install, upgrade, and uninstall itself for any chat, without a signed-in user, and manage its permission grants for accessing those specific chats data.| NA | Supported |
```
Comment (AkJo) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:104]: This is not an RSC permission. We should not be adding it here.

Comment (v-bvishnu) [reply msteams-platform/graph-api/rsc/resource-specific-consent.md:104]: Hi @AkJo: Request you to let me know where exactly to be added.

**Comment by v-bvishnu** [comment]
TeamsAppInstallation.ReadWriteAndConsentSelfForChat.All API permission is already available, 

live site: [List apps in chat - Microsoft Graph v1.0](https://learn.microsoft.com/en-us/graph/api/chat-list-installedapps?view=graph-rest-1.0&tabs=http#permissions)

**Pull request closed without merging**